src/Console/Concerns/ChecksDocker.php

```
@@ -0,0 +1,295 @@
+        return 'unknown';
+    }
+
+    /**
+     * Check if a command is available.
+     *
+     * @param string $command
+     * @return bool
+     */
+    protected function commandAvailable(string $command): bool
+    {
+        exec("which {$command} 2>/dev/null", $output, $returnCode);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)